### DISCUSSION TITLE
Companion & ATEM

### DISCUSSION THREAD

**Discussion by coxnb** [bitfocus/companion#2496] [Q&A] [answered]
Need help creating a "Fade/Cut" button in Companion that will communicate with the ATEM to fade and cut to the previewed camera. Currently, we are utilizing JustMacros to execute this action in Companion. I'm unfamiliar with JustMacros and Lua coding. 

I attempted to create this Fade/Cut button in Companion before posting this; no success. Would it be simpler to record a fade/cut macro on the ATEM and select that in Companion or is there something I'm missing? Thanks.

**Comment by Julusian** [comment]
do you want it to fade or to cut? those are different actions.

Doing a fade is as simple as adding an action to a button:
![image](https://github.com/bitfocus/companion/assets/1327476/cb59530c-4cb0-4f1a-bda7-171cb62a3c6b)
This is assuming you have the transition on the atem set as a mix with the correct rate. If you don't then there are other actions which will set that up

**Reply by coxnb** [reply]
Apologies, in this scenario I want it to cut to previewed camera

**Reply by Julusian** [reply, marked as answer]
Then use the `ME: Perform CUT transition` action instead

**Reply by coxnb** [reply]
That worked, thanks for your help!
